conf/config-default.yaml

```diff
@@ -114,9 +114,10 @@ apisix:
     ssl_session_tickets: false              #  disable ssl_session_tickets by default for 'ssl_session_tickets' would make Perfect Forward Secrecy useless.
                                             #  ref: https://github.com/mozilla/server-side-tls/issues/135
 
-    key_encrypt_salt: edd1c9f0985e76a2      #  If not set, will save origin ssl key into etcd.
-                                            #  If set this, must be a string of length 16. And it will encrypt ssl key with AES-128-CBC
-                                            #  !!! So do not change it after saving your ssl, it can't decrypt the ssl keys have be saved if you change !!
+    key_encrypt_salt:             #  If not set, will save origin ssl key into etcd.
+      - edd1c9f0985e76a2          #  If set this, must be a string of length 16. And it will encrypt ssl key with AES-128-CBC
+                                  #  !!! So do not change it after saving your ssl, it can't decrypt the ssl keys have be saved if you change !!
+                                  #  Only use the first key to encrypt, and decrypt in the order of the array.
```

Comment:
The comment is not accurate to describe the field, since the comment says it "must be a string of length 16".

Reply:
Done

Reply:
The comment is still wrong.

The `key_encrypt_salt` should either be a string whose size is `16` or an array whose elements are string, and the size is also `16`.

Reply:
I'm updating this to `a hexadecimal string of length 16`. It's technically a 8 byte value in hex representation. Size isn't accurate.